src/components/Reports/ProjectReport/PiechartByProject/PieChartByProject.css

```diff
@@ -1,7 +1,15 @@
 .pie-chart-title {
-  text-align: center;
-  margin-bottom: 24px;
-  color: black; /* Default for light mode */
```

Comment:
Need to include this line in the code for light mode, If I check in light mode, pie-chart banner is vanish.
because above line has been removed from the code.
( color: black; /* Default for light mode */ )

Reply:
![Screenshot 2024-12-07 at 7 16 07 PM](https://github.com/user-attachments/assets/ab5c8455-2436-4630-90c5-3555bd91c604)

Hey @ShraddhaShahari, the Pie Charts text is visible in both dark and light mode. Also visible, when we switch from dark to light mode